x/ibc/core/04-channel/keeper/packet.go

```diff
@@ -104,9 +104,14 @@ func (k Keeper) SendPacket(
 		)
 	}
 
+	commitment, err := types.CommitPacket(k.cdc, packet)
+	if err != nil {
+		return sdkerrors.Wrap(types.ErrInvalidPacket, err.Error())
```

Comment:
```suggestion
		return sdkerrors.Wrap(err, "failed to create packet commitment from packet)
```
nit: I think something like this provides more context